src/EventAction.cxx

```diff
@@ -250,7 +250,7 @@ void EventAction::ReOrderTrackIds(Int_t subId) {
         for (int n = 0; n < restG4Event->GetNumberOfTracks(); n++) {
             const auto& track = restG4Event->GetTrack(n);
             if (track.GetSubEventID() == subId - 1) {
-                if (track.isRadiactiveDecay()) {
+                if (track.ContainsProcess(6210) /* Decay, verified on v11.0.2 */) {
```

Comment:
Would not be better to re-implement `isRadiactiveDecay`? Looks not good hardcoding the process id like that, specially with the fact that those ids could be changing as I understood from our meeting yesterday.

Reply:
Yes, this was only a temporary change (remember the PR is still on draft), it has been addressed now, but thanks for the feedback!